oxidation/libparsec/crates/client_connection/Cargo.toml

```
@@ -4,6 +4,11 @@ version = "0.1.0"
 edition = "2021"
 autotests = false
 
+[features]
+test-testbed-support = [
```

```suggestion
testbed-support = [
```

the `test` prefix is redundant for me

---

You test the testbed, it's not redundant :thinking: